### PR TITLE
Fix: WINDOW MORRESS CALCULATION does not correctly show records

### LINKED ISSUE #1133: [Bug Report] Ventana Cálculo de Morosidad no muestra correctamente los registros
## Bug report

La ventana Cálculo de Morosidad no muestra los registros correctamente, no permite filtrar entre los registros de Entrada

#### Steps to reproduce

1. Abrir ventana de Cálculo de Morosidad
2. Intentar navegar entre los registros de la entrada

#### Screenshot or Gif（截图或动态图）

image

### PR DESCRIPTION
**Pull request by elsiosanchez**
Peek 2023-06-20 14-39

Fix: https://github.com/solop-develop/frontend-core/issues/1133

### CODE REVIEW

File: components/ADempiere/PanelInfo/Component/RecordLogs/index.vue [hunk 1]
```diff
@@ -212,9 +212,16 @@ export default defineComponent({
       return isDocumentStatus({ columnName: changeLog.columnName })
     }
 
+    const getTableName = computed(() => {
+      const { currentTab } = store.getters.getContainerInfo
```
Comment (EdwinBetanc0urt) [inline components/ADempiere/PanelInfo/Component/RecordLogs/index.vue]: please change by `getStoredTableNameByTab` getter

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

**Comment by elsiosanchez** [comment]
Jodase menor

**Comment by elsiosanchez** [comment]
eso quedara asi

File: components/ADempiere/PanelInfo/Component/RecordLogs/index.vue (hunk 1 above)
Comment (elsiosanchez) [reply components/ADempiere/PanelInfo/Component/RecordLogs/index.vue]: https://github.com/solop-develop/frontend-default-theme/pull/697/commits/27264ca6a119fe1d0581ccfa9f2f01976e566e75

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**